### PR TITLE
[WebUI][test][browser] Use HAPI2 plugins as test targets.

### PR DESCRIPTION
**Pull request by kz0817**
This patch is one of preparations for addressing #1938, which
will remove legacy built-in Arms for Zabbix and nagios.
So the tests using them should be fixed to use those of HAPI2
version.

### CODE REVIEW

**Comment by kz0817** [comment]
wercker test failed? I'll check it soon.

**Comment by cosmo0920** [comment]
Fetching repository metadata step is failed.
We should restart it.
Wercker CI is only used for building RPMs.

**Comment by kz0817** [comment]
> Fetching repository metadata step is failed.

So it was due to the wercker's system and the patch has no problem, wasn't it ?
And I could make RPM with this branch on my local machine.

Please continue to review it.

File: client/test/browser/test_events_view.js [hunk 1]
```diff
@@ -432,7 +451,7 @@ describe('EventsView', function() {
     var configJson =
       '{"events.columns":"eventId,' +
       'incidentStatus,incidentPriority,incidentAssignee,incidentDoneRatio"}';
-    respond(eventsJson(events, getDummyServerInfo(0)),
+    respond(eventsJson(events,getDummyHAPI2ZabbixServerInfo()),
```
Comment (cosmo0920) [inline client/test/browser/test_events_view.js]: Nit: Please add a space before `getDummyHAPI2ZabbixServerInfo()`.

File: client/test/browser/test_events_view.js [hunk 2]
```diff
@@ -455,8 +474,7 @@ describe('EventsView', function() {
     var configJson =
       '{"events.columns":"eventId,' +
       'incidentStatus,incidentPriority,incidentAssignee,incidentDoneRatio"}';
-    respond(eventsJson(events, getDummyServerInfo(0)),
-            configJson);
+    respond(eventsJson(events,getDummyHAPI2ZabbixServerInfo()), configJson);
```
Comment (cosmo0920) [inline client/test/browser/test_events_view.js]: Ditto.

**Comment by cosmo0920** [comment]
Looks good, except for a few style nitpicks. :smiley:

**Comment by kz0817** [comment]
I fix the lines pointed out.

**Comment by cosmo0920** [comment]
Bravo!

**Comment by cosmo0920** [comment]
Any concerns?
I'll merge this PR tomorrow.

**Comment by ashie** [comment]
:+1:

**Pull request merged**